metrocosm/ipaths.js

```diff
@@ -384,9 +435,44 @@ return my_array
       .style("stroke","rgb(255,255,255)")
       .style("stroke-width",0.5)
       .style("fill", function(d) {
-           if (d.properties.adm0_a3 == "USA") {return "black";}
-           if (d.properties.adm0_a3 == "RUS") {return "rgb(138,43,226)";}
-           return colors[d.properties.mapcolor13];
+           // if (d.properties.adm0_a3 == "USA") {return "black";}
+           // if (d.properties.adm0_a3 == "RUS") {return "rgb(138,43,226)";}
+           // return colors[d.properties.mapcolor13];
+            if (d.properties.adm0_a3 == "AFG") {return "blue";}
+			if (d.properties.adm0_a3 == "ARM") {return "blue";}
```

Comment:
You may want to create a new map array (country_color_map) and use it to look up instead